src/Sulu/Bundle/HttpCacheBundle/EventSubscriber/InvalidationSubscriber.php

```diff
@@ -180,7 +180,7 @@ public function invalidateDocumentBeforeRemoving(RemoveEvent $event)
     /**
      * Invalidates the structure of the given document.
      *
-     * @param $document
+     * @param object $document
```

Comment:
as far as i see, all calls to this method look like this:
```
        if ($document instanceof StructureBehavior) {
            $this->invalidateDocumentStructure($document);
        }
```

so i think we should use `StructureBehavior`
```suggestion
     * @param StructureBehavior $document
```

Reply:
@nnatter a document can be any object so `object` is correct here.